js/data.js

```diff
@@ -344,12 +388,23 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze", "Frank Feinbube", "Uwe Hentschel"],
         nameLV: "Eingebettete Betriebssysteme"
     },
+    eingebetteteBetriebssystemeWithITSE: {
+        kurz: "Embedded<br />Systems (mit ITSE)",
+        kennung: [IST, OSIS, SAMT, ITSE],
```

Comment:
I do not find that information on that site.
Even if, we would need to follow the official document, so for now I think its safer to remove `SAMT`.